src/EnergyPlus/DXCoils.hh

```diff
@@ -416,7 +402,7 @@ namespace DXCoils {
         Real64 CurrentEndTimeLast; // end time of time step for last simulation time step
         Real64 TimeStepSysLast;    // last system time step (used to check for downshifting)
         // for multispeed DX coil type
-        int FuelType;                               // Fuel type
+        std::string FuelType;                       // Fuel type
```

Comment:
@dareumnam I think we need two variables here - `int` and `std:string` and use `ValidateFuelTypeWithAssignResourceTypeNum` to set them both during input processing. The string is needed to build output variable names, but the int should be used in all of the if's, or at least in any if's that are triggered during the simulation. We don't want any string compares during the simulation.

Reply:
I got it. I'll update the code as your comment.